lib/trello_flow/pull_request.rb

```diff
@@ -22,11 +22,19 @@ def url
       end
 
       def title
-        card.name.gsub('"',"'") + " [Delivers ##{card.short_link}]"
+        card_name + " [Delivers ##{card.short_link}]"
+      end
+
+      def card_name
+        card.name.gsub('"',"'")
```

Comment:
Suggestion: How about use the char syntax: `card.name.gsub(?", ?')` or use `%()`.